src/string.cr

```diff
@@ -3627,37 +3627,61 @@ class String
   # "Aubergine".rjust(8)   # => "Aubergine"
   # ```
   def rjust(len, char : Char = ' ')
-    just len, char, false
+    just len, char, 1
   end
 
-  private def just(len, char, left)
+  # Adds instances of *char* to left ond right of the string until it is at least size of *len*.
+  #
+  # ```
+  # "Purple".center(8)      # => " Purple "
+  # "Purple".center(8, '-') # => "-Purple-"
+  # "Purple".center(9, '-') # => "-Purple--"
+  # "Aubergine".center(8)   # => "Aubergine"
+  # ```
+  def center(len, char : Char = ' ')
+    just len, char, 0
+  end
+
+  private def just(len, char, justify)
     return self if size >= len
 
     bytes, count = String.char_bytes_and_bytesize(char)
-
-    difference = len - size
-    new_bytesize = bytesize + difference * count
+    padding = (len - size)
+    new_bytesize = bytesize + padding * count
+    case justify
+    when .< 0
+      leftpadding, rightpadding = 0, padding
+    when .> 0
+      leftpadding, rightpadding = padding, 0
+    else
+      leftpadding = padding >> 1
```

Comment:
`// 2`, please

there's no need for manual pinhole optimizations the compiler is perfectly capable of.

Reply:
Ok, done !